language_detector/main.py

```diff
@@ -1,9 +1,16 @@
 """This is the entry point of the program."""
-
+# coding=utf-8
 from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    spanish, german, english = [set(LANGUAGES[x]['common_words']) for x in range(3)]
+    language_list = [spanish, german, english]
```

Comment:
I agree with you. Hard coding parameters is not a good idea. Will definitely look for a solution in that direction. Thanks for your suggestion!